test/Directory.Packages.props

```diff
@@ -0,0 +1,138 @@
+<Project>
+  <Import Project="..\Directory.Packages.props" />
+
+  <!-- TODO: I'm having second thought on whether it would be better to consolidate all packages in one ItemGroup and sort them alphabetically. -->
+  <!-- TODO: Take special care for the complicated logic in TestApplication.MongoDB.csproj -->
+
+  <!-- Versions from Directory.Build.props in the same folder -->
+  <ItemGroup>
+	<PackageVersion Include="FluentAssertions" Version="6.9.0" />
+    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
+    <PackageVersion Include="Moq" Version="4.18.4" />
+    <PackageVersion Include="xunit" Version="2.4.2" />
+    <PackageVersion Include="xunit.runner.visualstudio" Version="2.4.5" />
+  </ItemGroup>
+
+  <!-- Versions from IntegrationTests.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="Google.Protobuf" Version="3.21.12" />
+    <PackageVersion Include="Grpc.Tools" Version="2.51.0" />
+    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.0.1" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageVersion Include="StrongNamer" Version="0.2.5" />
+    <PackageVersion Include="System.Collections.Immutable" Version="7.0.0" />
+    <PackageVersion Include="System.Runtime.InteropServices" Version="4.3.0" />
+    <PackageVersion Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <PackageVersion Include="Testcontainers" Version="2.3.0" />
+    <PackageVersion Include="Verify.Xunit" Version="19.7.0" />
+    <PackageVersion Include="xunit.abstractions" Version="2.0.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
+    <PackageVersion Include="Microsoft.AspNetCore.Hosting" Version="2.2.7" />
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.Shared.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="System.ValueTuple" Version="4.5.0" />
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.Elasticsearch.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="Elastic.Clients.Elasticsearch" Version="8.0.0" />
+  </ItemGroup>
+  
+  <!-- Versions from TestApplication.GraphQL.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="GraphQL" Version="2.3.0" />
+    <PackageVersion Include="GraphQL.StarWars" Version="1.0.0" />
+    <PackageVersion Include="GraphQL.Server.Transports.AspNetCore" Version="3.4.0" />
+    <PackageVersion Include="GraphQL.Server.Ui.Playground" Version="3.4.0" />
+    <PackageVersion Include="Microsoft.AspNetCore" Version="2.2.0" />
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.GrpcNetClient.csproj -->
+  <ItemGroup>
+    <!-- TODO: VersionOverride: reason unknown for why this csproj used a lower version. Need to verify if latest version works. -->
```

Comment:
It is outside the scope of this PR, use the version utilized in the project.

Reply:
Or leave `TestApplication.GrpcNetClient.csproj` out of centralization.

Reply:
Done.